sklearn/feature_extraction/_hash.py

```diff
@@ -53,7 +53,7 @@ class FeatureHasher(TransformerMixin, BaseEstimator):
         The number of features (columns) in the output matrices. Small numbers
         of features are likely to cause hash collisions, but large numbers
         will cause larger coefficient dimensions in linear learners.
-    input_type : {"dict", "pair", "string"}, default="dict"
+    input_type : {"dict", "pair", "str"}, default="dict"
```

Comment:
By looking at the description below, this should be `"string"` and not `"str"`. Indeed, this is not the Python type but a "string" as well.